docs/web/sdk-services-commands-operations.mdx

```diff
@@ -50,17 +50,77 @@ export default class ExampleService extends ServiceBase {
 
 ## Implementing Commands and Operations
 
-Services can also register an implementation for any command or operation. The following service registers an implementation for a custom command that takes a `string` argument.
+Services can also register an implementation for any command or operation. Services can either register a command or operation by a static reference to a function, or by dynamically registering the implementation at runtime.
```

Comment:
Services must *not* use `.register()`. That is an anti-pattern. Dynamic registration is only intended for components. The callback would only ever get invoked if the service happened to be injected as a required dependency, or explicitly configured as a startup service. Otherwise it would never be invoked since the framework would have no reason to create the service in the first place. The confusion around this is precisely why I wanted to overhaul command registration :-)

Reply:
Ahh... I see. Yes, this is a perfect demonstration of that confusion ahahaha.

Removed the anti-pattern.